frontend/src/lib/components/widgets/TextInput/TextInput.test.tsx

```diff
@@ -226,28 +226,6 @@ describe("TextInput widget", () => {
     expect(wrapper.find(UIInput).prop("value")).toBe("0123456789")
   })
 
-  it("updates widget value on text changes when inside of a form", () => {
```

Comment:
Why can we remove this test? Isn't the text input / text area value still kept in sync between the value in the input field and the widget manager value?

Reply:
In `TextInput.tsx` and `TextArea.tsx`, I removed the committing of value and made sure the value is always dirty when it's changed. The reason was because when you're in a form and we commit the value, then the value is not dirty and therefore the `Press Enter to submit form` text won't be shown.